pages/people.md

```diff
@@ -139,6 +139,7 @@ GitHub also has a [contributor list][github-contributors] based on commits.
 | Volth               |                                                                   |                       |
 | Wil Selwood         | [Satellite Applications Catapult][SACAT]                          | [GMT][GMT]/[BST][BST] |
 | Kyle Van Gilson     |                                                                   |                       |
+| Jeffrey Zeiberg     | [Arctic Slope Regional Corp.][ASRC]                               | [ET][ET]              |
```

Comment:
This breaks the sort order :smiley_cat:

Reply:
The previous entry does also.  I can fix them up when I publish the page.